modules/update-upgrading-cli.adoc

```diff
@@ -75,19 +61,22 @@ VERSION IMAGE
 4.10.12 quay.io/openshift-release-dev/ocp-release@sha256:f77f4f75c1e1a4ddd0a0355f298a834db3473fd9ca473235013e9419d1df16db
 4.10.13 quay.io/openshift-release-dev/ocp-release@sha256:4f516616baed3cf84585e753359f7ef2153ae139c2e80e0191902fbd073c4143
 ----
+[NOTE]
+*For details and information on how to perform an `EUS-to-EUS` channel upgrade, please refer to the 
+_Preparing to perform an EUS-to-EUS upgrade page_, listed in the Additional resources section.
 
-. Based on your organization requirements, set the upgrade channel to `stable-{product-version}`, `fast-{product-version}`, or `eus-{product-version}`:
+. Based on your organization requirements, set the appropriate upgrade channel. For example, you can set your channel to `stable-4.9`, `fast-4.9`, or `eus-*`. For more information about channels, refer to _Understanding update channels and releases_ listed in the Additional resources section.
 +
 [source,terminal]
 ----
 $ oc adm upgrade channel <channel>
 ----
 +
-For example, to set the channel to `stable-{product-version}`:
+For example, to set the channel to `stable-4.9`:
 +
-[source,terminal,subs="attributes+"]
+[source,terminal]
 ----
-$ oc adm upgrade channel stable-{product-version}
+$ oc adm upgrade channel stable-4.9
```

Comment:
The channel version in the examples looks not that good to me. The default channel on a 4.9 cluster is stable-4.9. I'd prefer to have stable-4.9 as the example channel in procedure#1 and switch to stable-4.10 in the step.

Reply:
Hi @shellyyang1989 ! Sounds good :) I assume this also changes the command 
`$ oc adm upgrade channel stable-4.9` to `$ oc adm upgrade channel stable-4.10` on line 82?